bin/node/runtime/src/lib.rs

```diff
@@ -2166,4 +2166,35 @@ mod tests {
 			size,
 		);
 	}
+
+	#[test]
+	fn check_pallet_storage_sizes() {
+		use frame_support::traits::StorageInfoTrait;
+		let mut storage_info = AllPalletsWithSystem::storage_info();
+		println!("| Pallet | Storage | Max Values | Max Size |");
```

Comment:
nit: Maybe some formatting for readability
```suggestion
		println!(
			"| {:^30} | {:^30} | {:^10} | {:^15} |",
			"Pallet", "Storage", "Max Values", "Max Size"
		);
```